Earthfile

```diff
@@ -25,24 +25,24 @@ sbt-deps:
 
 npm-deps:
   COPY web/package.json web/yarn.lock web/
-  RUN cd web; yarn --frozen-lockfile
+  RUN cd web && yarn --frozen-lockfile
```

Comment:
Good idea!

Reply:
> Good idea!

For `cd` it doesn't make much difference. But I used `;` instead of `&&` at some other step and ended up ignoring a failure as a result. I took revenge on all semicolons.